### ISSUE TITLE
Generated files should be marked as auto-generated to avoid Code Analysis errors

### ISSUE THREAD

**Issue by SteveDunn** [progala2/Buildenator#38]
I just added a builder to my test project and ended up with hundreds of code analysis errors.

The generated code should include a header to say that it's generated. Here's [an example](https://github.com/SteveDunn/Vogen/blob/main/src/Vogen/ValueObjectGenerator.cs):

```csharp
// ------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a source generator named Vogen (https://github.com/SteveDunn/Vogen)
//
//     Changes to this file may cause incorrect behavior and will be lost if
//     the code is regenerated.
// </auto-generated>
// ------------------------------------------------------------------------------
```

**Comment by progala2** [comment]
thanks, I'll do it this week!

**Comment by progala2** [comment]
Ok, Done it right away. Included in the 4.1.4, should appear in 15 minutes on nuget.

**Comment by SteveDunn** [comment]
That was quick! Many thanks
